gammapy/irf/effective_area.py

```diff
@@ -116,6 +117,27 @@ def to_table(self):
         meta.update(HI_THRES=self.high_threshold.to('TeV').value)
         return Table([ener_lo, ener_hi, self.data], names=names, meta=meta)
 
+    def area_max(self, percent_area_max):
```

Comment:
I would remove the connection to the energy threshold here (This is related to spectral fitting not to the effective area intrinsically). Why not add a property
```
@property
def area_max(self)
"""Maximum effective area"""
```
This can be useful in any context

And then another method
```
def find_energy(effective_area)
"""Return energy for a given effective area value"""
```
If you use this function to calculate a threshold you can then just type
```
threshold = effarea.find_energy(0.1 * effarea.area_max)
```
I find this very well readable